actions/plugins/publish/change-plugin-scope/action.yml

```diff
@@ -0,0 +1,52 @@
+name: Change plugin scope
+
+inputs:
+  gcom-token:
+    description: GCOM token with permissions to update a plugin version.
+    required: true
+  gcloud-auth-token:
+    description: |
+      Google Cloud auth token for IAP access.
+      Required only for to dev or ops access.
+    required: false
+    default: ""
+  environment:
+    description: |
+      Environment where the plugin is published.
```

Comment:
we should clarify this. this is not exactly the environment where the plugin ispublished but the environment in which this workflow will change the scope.